log/slow/parser.go

```diff
@@ -33,6 +33,7 @@ import (
 // Regular expressions to match important lines in slow log.
 var timeRe = regexp.MustCompile(`Time: (\S+\s{1,2}\S+)`)
 var userRe = regexp.MustCompile(`User@Host: ([^\[]+|\[[^[]+\]).*?@ (\S*) \[(.*)\]`)
+var schema = regexp.MustCompile(`Schema: (.+) Last_errno:`)
```

Comment:
The regexp should be:
```
var schema = regexp.MustCompile(`Schema: *(\S+) *Last_errno:`)
```
to not to have spaces in the schema name. This also fixes all the tests:
```
 go test
OK: 22 passed
PASS
ok      github.com/percona/go-mysql/log/slow    0.008s
```